### ISSUE TITLE
Google Keep Clone

### ISSUE THREAD

**Issue by Goheljay** [hash-define-organization/Hacktober-Fest-2021#195]
I would like to make a Similar Google keep clone using HTML, CSS, JS for Hacktoberfest, Please assign me.

**Comment by WINOFFRG** [comment]
Share more details if it would be using a Database or not. Also, share and preview images. Assigned!

**Comment by Goheljay** [comment]
No, it does not use any database but uses the local storage of your browser.

Screen shot
![google keep](https://user-images.githubusercontent.com/54993657/135961232-989ed107-3462-42fa-b6a6-72875e967814.JPG)

**Comment by ij007** [comment]
Closing the issue as PR got merged.
